plugin/src/main/java/io/deephaven/plugin/type/ObjectType.java

```diff
@@ -99,4 +99,20 @@ interface Reference {
             Optional<String> type();
         }
     }
+
+    void addMessageSender(MessageSender channel);
+
+    void removeMessageSender();
+
+    void handleMessage(String message);
```

Comment:
To match the OutputStream, probably want to make this InputStream? Also, it should reference the tickets that the client sent to the server as a second arg (probably list of Object?).

Reply:
It's now just `byte[]` and also the objects that the tickets reference from the client. If you still think this should be a stream, can you give an example of how that might look? The main thing I'm concerned about is getting it into the Python side in a way that is at least somewhat user friendly. There is a layer (ObjectTypeAdapter in __init__.py) where we can do not user friendly things though if needed